packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -180,6 +184,8 @@ export const Tooltip = ({
     );
   }
 
+  const delay = !hideDelay ? (closeOnMouseLeave ? 0 : 300) : hideDelay;
+
```

Comment:
When @Miretxu introduced the behavior for a user to keep the tooltip open, we didn't think about allowing them to set different delays and that's why we have the `closeOnMouseLeave` prop

now that we have the custom delay functionality, it looks to me that we could deprecate that prop and simply pass hideDelay to the setTimeout like this: 
 
```jsx
function Tooltip (
  hideDelay = 0 // setting the default to 0 because the default behavior is not having any delay, right?
  // ...
)

// inside the return...
<ContainerElement
  onMouseLeave={(evt: MouseEvent) => {
    setTimeout(() => setIsHoveringTarget(false), hideDelay);
    if (onMouseLeave) onMouseLeave(evt);
   }
>
// rest of the jsx...
```

so a user that wants the tooltip to close immediately after not hovering the target would do:
`<Tooltip />`

and a user that wants the Tooltip to stay open would do this:
`<Tooltip hideDelay={300} />`

I think this is simpler than having all the logic mixing two props that are essentially doing the same. My question to you and @Miretxu now is: are you ok with deprecating `closeMouseOnLeave`? (I believe your team is the only ones using it right now)

Reply:
Sorry for keep commenting on your PR, we are trying to keep components easy and simple to use. But sometimes solutions take time to show up 🙏 

If you want to pair on this ping me :)

Reply:
On a BIG sidenote here

I suggested in the comment above to set default value for the prop in the argument of the function, but this component is still using the old way of setting default props in line 245 (Component.defaultProps = { bla bla bla })

The forma36 team decided to refactor this, so it would be a HUGE help if you use this PR to move all the defaults to the argument of the function, but not necessary

Reply:
@gui-santos happy to deprecate it to make the component simpler and cleaner 👍
We can keep an example in the storybook to demonstrate how to use hideDelay to achieve the content hover-over.

If we remove `closeMouseOnLeave` props, we can set the hideDelay default to 0 in `Component.defaultProps = { }`.
 Previously, we had to calculate it either 0 or 300 depending on the `closeMouseOnLeave` value, and the prop value wasn't available in `Component.defaultProps = { }`, that's why we defined in the argument of the function. But we won't have that problem now 🙂

Reply:
@gui-santos I wonder if we should the default `hideDelay` to 0 or 300? 300 will enable the content hover over but not sure if it will introduce "drags" to the UI for a small tooltip, like a copy-paste button.

Reply:
@gui-santos Thank you very much for your suggestions! Me and @Miretxu just addressed all the comments and refactored the component as recommended. Could you please have a look again?